azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/serializer/V2ConfigurationSerializer.java

```diff
@@ -23,9 +23,10 @@ public V2ConfigurationSerializer(WebAppConfiguration newConfigs, WebAppConfigura
     }
 
     @Override
-    public void saveToXML(Element configurationElement)
+    public void saveToXML(Element configurationElement, String subscriptionId)
         throws MojoFailureException {
         createOrUpdateAttribute("schemaVersion", "V2", oldConfigs.getSchemaVersion(), configurationElement);
+        createOrUpdateAttribute("subscriptionId", subscriptionId, null, configurationElement);
```

Comment:
creating `<subscription>` node directly may override users' parameters, like `<subscriptionId>${subs}</subscriptionId>`, so I prepare to use `createOrUpdateAttribute` to compare with previous configuration and save only if the value changed